docs/en/latest/plugins/ua-restriction.md

```diff
@@ -39,11 +39,11 @@ A common scenario is to set crawler rules. `User-Agent` is the identity of the c
 | bypass_missing | boolean       | False    | false        |                         | When set to `true`, bypasses the check when the `User-Agent` header is missing. |
 | allowlist      | array[string] | False    |              |                         | List of allowed `User-Agent` headers.                                           |
 | denylist       | array[string] | False    |              |                         | List of denied `User-Agent` headers.                                            |
-| message        | string        | False    | "Not allowed" | [1, 1024] | Message with the reason for denial to be added to the response.                 |
```

Comment:
`[1, 2024]` was removed because it was confusing. `message` is a string type, and yet, the valid value was given in numeric range.

Upon checking the code, the numeric range meant the min and max length:

![image](https://github.com/apache/apisix/assets/39619599/318790f9-30d7-4a51-8e06-e3871830fea2)

i think this can just be removed.